types/meyda/index.d.ts

```diff
@@ -0,0 +1,137 @@
+// Type definitions for meyda 4.2
+// Project: https://github.com/meyda/meyda
+// Definitions by: Damien Erambert <https://github.com/eramdam>
+//                 Hugh Rawlinson <https://github.com/hughrawlinson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export type MeydaWindowingFunction = 'blackman' | 'sine' | 'hanning' | 'hamming';
+export type MeydaAudioFeature =
+    | 'amplitudeSpectrum'
+    | 'chroma'
+    | 'complexSpectrum'
+    | 'energy'
+    | 'loudness'
+    | 'mfcc'
+    | 'perceptualSharpness'
+    | 'perceptualSpread'
+    | 'powerSpectrum'
+    | 'rms'
+    | 'spectralCentroid'
+    | 'spectralFlatness'
+    | 'spectralFlux'
+    | 'spectralKurtosis'
+    | 'spectralRolloff'
+    | 'spectralSkewness'
+    | 'spectralSlope'
+    | 'spectralSpread'
+    | 'zcr'
+    | 'buffer';
+
+export type MeydaSignal = SliceableArrayLike<number>;
+export interface SliceableArrayLike<T> extends ArrayLike<T> {
+    slice(start: number, end: number): SliceableArrayLike<T>;
+}
+
+type MeydaSource = AudioNode;
+
+export interface MeydaFeaturesObject {
+    amplitudeSpectrum: Float32Array;
+    buffer: number[];
+    chroma: number[];
+    complexSpectrum: {
+        real: number[];
+        imag: number[];
+    };
+    energy: number;
+    loudness: {
+        specific: Float32Array;
+        total: number;
+    };
+    mfcc: number[];
+    perceptualSharpness: number;
+    perceptualSpread: number;
+    powerSpectrum: Float32Array;
+    rms: number;
+    spectralCentroid: number;
+    spectralFlatness: number;
+    spectralKurtosis: number;
+    spectralRolloff: number;
+    spectralSkewness: number;
+    spectralSlope: number;
+    spectralSpread: number;
+    zcr: number;
+}
+
+export class MeydaAnalyzer {
+    start(features: MeydaAudioFeature[]): void;
+
+    stop(): void;
+
+    setSource(source: MeydaSource): void;
+
+    get(features: MeydaAudioFeature[]): Partial<MeydaFeaturesObject> | null;
+}
+
+interface Meyda {
```

Comment:
Instead of and interface you should just export each property individually so that it's available on the namespace context. e.g:
```js
export const audioContext: AudioContext | null;
...
export function windowing(signal: MeydaSignal, windowname?: MeydaWindowingFunction): MeydaSignal;
...
```